client/index.tpl.html

```diff
@@ -58,6 +60,14 @@
         <script src="bower_components/cryptojs/lib/PBKDF2.js"></script>
         <script src="bower_components/cryptojs/lib/HMAC.js"></script>
         <script src="bower_components/cryptojs/lib/AES.js"></script>
+
+        <script src="bower_components/jquery-ui/jquery-ui.js"></script>
```

Comment:
Does `wiredep` task gives you already `jquery-ui`?

Reply:
strange. Take a look at https://github.com/ovh-ux/ovh-manager-telecom/blob/11534aaeb6efec93d4d18fcab1a96ae0c2be69ca/bower.json#L137. We override jquery-ui dependency with the plugins we need. 
Maybe you need another plugin to be loaded. If so feel free to modify bower.json :-)

Reply:
Thanks a lot! It's fixed now. couple of draggable dependencies were missing ;)